spec/Vehicle/CrashManagement.vspec

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2024 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+
+CrashManagement:
```

Comment:
It can be discussed if we want to have the signals on top "Vehicle" level, in "Body" or if we want a separate branch for signals related to crash prevention and crash detection. Exact definition/meaning of signals can also be discussed